### ISSUE TITLE
Import Media Json fails

### ISSUE THREAD

**Issue by r0uv3n** [IgnisDa/ryot#412]
Importing a recently exported `ryot_export.json` file via the "Media Json" Importer fails with the following appearing in logs:
```
thread 'tokio-runtime-worker' panicked at apps/backend/src/importer/media_json.rs:6:53:
called `Result::unwrap()` on an `Err` value: Error("invalid type: map, expected a sequence", line: 1, column: 0)
stack backtrace:
   0:     0x7f36886676c1 - <unknown>
   1:     0x7f3687753c3c - <unknown>
   2:     0x7f368866425e - <unknown>
   3:     0x7f36886674a4 - <unknown>
   4:     0x7f368866e383 - <unknown>
   5:     0x7f368866e0cf - <unknown>
   6:     0x7f368866e876 - <unknown>
   7:     0x7f368866e784 - <unknown>
   8:     0x7f368866d2d6 - <unknown>
   9:     0x7f368866e512 - <unknown>
  10:     0x7f368738e1e3 - <unknown>
  11:     0x7f368738e5ca - <unknown>
  12:     0x7f368830e230 - <unknown>
  13:     0x7f36882deafe - <unknown>
  14:     0x7f3687fe1ecb - <unknown>
  15:     0x7f3687faaaa5 - <unknown>
  16:     0x7f3688295a81 - <unknown>
  17:     0x7f3688294643 - <unknown>
  18:     0x7f3688204b62 - <unknown>
  19:     0x7f36881e325b - <unknown>
  20:     0x7f3687facab1 - <unknown>
  21:     0x7f3687c9d441 - <unknown>
  22:     0x7f36883c0cf7 - <unknown>
  23:     0x7f3688698810 - <unknown>
  24:     0x7f3688698207 - <unknown>
  25:     0x7f36886a80f6 - <unknown>
  26:     0x7f36886aa01b - <unknown>
  27:     0x7f3688697bf3 - <unknown>
  28:     0x7f36886aa962 - <unknown>
  29:     0x7f36886a0669 - <unknown>
  30:     0x7f368869e4bc - <unknown>
  31:     0x7f36886af3c4 - <unknown>
  32:     0x7f3688695621 - <unknown>
  33:     0x7f3688694da3 - <unknown>
  34:     0x7f3688694cbf - <unknown>
  35:     0x7f36886995e8 - <unknown>
  36:     0x7f36886acc21 - <unknown>
  37:     0x7f36886a96d4 - <unknown>
  38:     0x7f36886af385 - <unknown>
  39:     0x7f368869eba7 - <unknown>
  40:     0x7f3688670fd5 - <unknown>
```
I have appended the [`ryot_export.json`](https://github.com/IgnisDa/ryot/files/13053931/ryot_export.json) file I used for testing this (I also formatted it for readability).

**Comment by IgnisDa** [comment]
You have made an export of the `all` section. You need to make it only for the `media` section. (https://ryot.fly.dev/export/media and not https://ryot.fly.dev/export/all).

**Comment by r0uv3n** [comment]
Ah yes that makes sense, thank you!

**Comment by r0uv3n** [comment]
I'm sorry, can we reopen this? I now get 
```
thread 'tokio-runtime-worker' panicked at apps/backend/src/importer/mod.rs:332:63:
called `Option::unwrap()` on a `None` value
```
I also tried deleting most of the `ImportOrExportMediaItem`s from the [`ryot_export.json`](https://github.com/IgnisDa/ryot/files/13054881/ryot_export.json) to isolate the issue, but it seems to happen for every entry.

**Comment by IgnisDa** [comment]
What version of ryot are you using?

**Comment by r0uv3n** [comment]
I have now also reproduced this on v3.0.4 (the unstable docker tag).

**Comment by IgnisDa** [comment]
Will be fixed by `v3.0.5` which will be released in around 40 minutes from now.

**Comment by IgnisDa** [comment]
I am curious, why do you need to export and then import into the same instance?

**Comment by r0uv3n** [comment]
I wrote a small Python script to convert a Simkl Export to Media JSON to migrate to Ryot, and the import failed, that's why I then tried to see if the import works for recently exported file. With your fix, everything works great. 

If you want, I can create an issue for the Simkl to Ryot Import and provide my script there. I thought about trying to write an importer like in [apps/backend/src/importer](https://github.com/IgnisDa/ryot/tree/main/apps/backend/src/importer) and creating a pull request afterwards, but I know neither Rust nor this code base, so the Python script was a lot quicker.
